libs/checkpoint/pyproject.toml

```diff
@@ -10,7 +10,7 @@ packages = [{ include = "langgraph" }]
 
 [tool.poetry.dependencies]
 python = "^3.9.0,<4.0"
-langchain-core = ">=0.2.22,<0.3"
+langchain-core = ">=0.2.38,<0.4"
```

Comment:
can we remove this dependency?

Reply:
don't think so, we're using it in jsonplus serializer

```python
from langchain_core.load.load import Reviver
from langchain_core.load.serializable import Serializable
```